plugin.json

```diff
@@ -6,13 +6,13 @@
     "posthogVersion": ">= 1.24.0",
     "config": [
         {
-          "key": "percentage",
-          "hint": "Reduces events flowing in to above %",
-          "name": "% of events to keep",
-          "type": "string",
-          "order": 1,
-          "default": "100",
-          "required": false
+            "key": "percentage",
+            "hint": "Reduces events flowing in to above %",
+            "name": "% of events to keep",
+            "type": "string",
+            "order": 1,
```

Comment:
Some context: This "order" is a leftover from an earlier time when we used config objects (key -> other props), not config arrays as we have now. Just to get rid of the annoying and hard to update `order: number` prop we switched to the arrays.

So can you remove it and also from other places in the future if you find them :)